collect_app/src/main/java/org/odk/collect/android/utilities/FormUtils.kt

```diff
@@ -0,0 +1,64 @@
+package org.odk.collect.android.utilities
+
+import org.javarosa.core.reference.ReferenceManager
+import org.javarosa.core.reference.RootTranslator
+import org.odk.collect.android.logic.FileReferenceFactory
+import org.odk.collect.android.storage.StoragePathProvider
+import org.odk.collect.forms.Form
+import java.io.File
+
+object FormUtils {
```

Comment:
Is there a reason we needed to do this in this PR? Maybe we can pull it out of the branch if not just so there's no extra risk added. If there's an advantage for the fix I'm missing I'm happy for it to stay in though!